.github/workflows/cre-system-tests.yaml

```diff
@@ -3,6 +3,11 @@ name: CRE System Tests
 on:
   workflow_dispatch:
     inputs:
+      ecr_name:
+        description: "The name of the ECR repository to push the image to, defaults to 'chainlink'"
```

Comment:
`pull the image from`?

Reply:
a8a3929c841abd43cda48d6065ae06d2a7018361